roles/mysql/defaults/main.yml

```diff
@@ -4,6 +4,7 @@
 
 ## mysql_root_password: $6$iiab51$3ICIW0CLWxxMW2a3yrHZ38ukZItD5tcadL4rWcE9D.qIGStxhh8rRsaSxoj3b.MYxI/VRDNjpzSYK/V6zkWFI0
 # mysql_root_password: fixmysql
+mysql_admin: mariaDBadmin
```

Comment:
Let's move this line down -- or put it in default_vars.yml ?

Either way: to respect the "# All above are set in: github.com/iiab/iiab/blob/master/vars/default_vars.yml" comment just below.

Reply:
It's just for quick testing.